frontend/src/app/header-bar/header-bar.component.ts

```diff
@@ -6,7 +6,6 @@ import { DomSanitizer } from '@angular/platform-browser';
 import { MatIconRegistry } from '@angular/material';
 import { MatSnackBar } from '@angular/material';
 import { CookieService } from 'ngx-cookie';
-import { AuthService } from '../shared/services/auth.service';
```

Comment:
So removing the AuthService is part of the overall task?

Reply:
explained in https://github.com/helm/monocular/pull/524#discussion_r220023390